albumentations/core/transforms_interface.py

```diff
@@ -48,20 +52,23 @@ def to_tuple(param, low=None, bias=None):
 @add_metaclass(SerializableMeta)
 class BasicTransform:
     call_backup = None
+    interpolation: Any
+    fill_value: Any
+    mask_fill_value: Any
 
-    def __init__(self, always_apply=False, p=0.5):
+    def __init__(self, always_apply: bool = False, p: float = 0.5):
         self.p = p
         self.always_apply = always_apply
-        self._additional_targets = {}
+        self._additional_targets: Dict[Any, Any] = {}
```

Comment:
Mypy gives me the following error when using the ```Dict[str, str]``` type hint.

```albumentations\core\transforms_interface.py:131: error: Argument 2 to "get" of "Mapping" has incompatible type "None"; expected "str"```

This is because the line below can return ```None``` if the key doesn't exist. What should I do here?

https://github.com/albumentations-team/albumentations/blob/2cefc162d7c605f76462e3cd5ce8c3c5117a44af/albumentations/core/transforms_interface.py#L122

Reply:
This is a very strange construction, because we already checked, that key exists. I think, we could change these lines:
```python
transform_key = key
if key in self._additional_targets:
    transform_key = self._additional_targets.get(key, None)
```
to these:
```python
transform_key = self._additional_targets.get(key, key)
```